gax/test/showcase-echo-client/README.md

```diff
@@ -9,4 +9,4 @@ It uses `google-gax`, so by having this generated code here, we can verify that
 changes made to this library don't break the code of the generated libraries.
 
 The only modification is in `package.json`, where it's set to depend on the local
-version of `google-gax` being tested.
+version of `google-gax` being tested.
```

Comment:
double check this for a whitespace error - I feel like it shouldn't have anything?

Reply:
added a new line